version/version_test.go

```diff
@@ -0,0 +1,22 @@
+package version
+
+import (
+	"regexp"
+	"testing"
+)
+
+func Test_EmptyVersionMeansBuildVersionReturnsDev(t *testing.T) {
+	Version = ""
+	output := BuildVersion()
+	if found, err := regexp.MatchString(`(?m:dev)`, output); err != nil || !found {
+		t.Fatalf("Output is not as expected:\n%s", output)
```

Comment:
Please change these to:

```
"message goes here - want: %s, got: %s"
```

Also remove the regex for strings.Contains or similar for readability.

Reply:
Fixed.